Lib/test/test_socket.py

```diff
@@ -4683,8 +4683,8 @@ def bind(self, sock, path):
                 raise
 
     def testUnbound(self):
-        # Issue #30205
-        self.assertEqual(self.sock.getsockname(), '')
+        # Issue #30205 (note getsockname() can return None on OS X)
```

Comment:
We are now using the "bpo-xxx" syntax instead of "issue #xxx" to prevent GitHub to creating a link to the GitHub bug tracker.